lib/deck.rb

```diff
@@ -0,0 +1,30 @@
+class Deck
+  attr_reader :cards 
+  def initialize(cards)
+    @cards = cards
+    # @deck = deck
+  end
+
+  def count
+  # def length
+  #   require "pry"; binding.pry
+    @cards.count
+  # end
+  # require "pry"; binding.pry
+  end
+  def cards_in_category(category)
+    # require "pry"; binding.pry
+    @cards.select do |card|
+      card.category == category
+  end
+    # matching_cards = []
+    # @cards.each do |card|
```

Comment:
Just as a general note, clean up your comments/binding.prys/code attempts before pushing up to main. It's okay to leave helpful comments for yourself, but it looks like you ended up commenting out old code which you didn't use, and you can go ahead and clean that up before merging into main.